docs/integration/KafkaIntegration.md

```diff
@@ -8,104 +8,164 @@ sidebar_position: 1
 
 To better understand how Nussknacker works with Kafka, it's recommended to read the following first:
 * [Kafka introduction](https://kafka.apache.org/intro)
-* [Role of Schema Registry](/about/TypicalImplementation)
+* [Role of Schema Registry](/about/TypicalImplementationStreaming)
 * [Confluent Schema Registry](https://docs.confluent.io/platform/current/schema-registry/index.html)
 
 If you want to use Flink engine, this is also recommended:
 * [Flink keyed state](https://ci.apache.org/projects/flink/flink-docs-master/docs/concepts/stateful-stream-processing/#keyed-state)
 
 ## Concepts
 
+### Sources and sinks
+
 Kafka topics are native streaming data input to Nussknacker and the native output where results of Nussknacker scenarios processing are placed. 
-In Nussknacker terminology input topics are called *sources*, output topics are called *sinks*. This section provides important details of Nussknacker's integration with Kafka and Schema Registry.
+In Nussknacker terminology input topics are handled by *source* components, output topics are handled by *sink* components.
+This section provides important details of Nussknacker's integration with Kafka and Schema Registry.
 
-## Schema Registry integration
+### Schemas
 
-Nussknacker integrates with Schema Registries. It is the source of knowledge about:
-- Which topics are available in Kafka sources and sinks
-- What schema versions are available for topic
-- How code completions and validations on types described by schemas should work
-- How messages will be serialized and deserialized
+Nussknacker uses schemas for code completions and validations of messages. 
```

Comment:
To me a bit too concise. What something like this 
-----
Schema defines the content and the format of data; Nussknacker expects that messages in the topics are described by the schema. Nussknacker uses information contained in schemas for code completion and validation of messages.

Reply:
Thanks for remark, but why content AND format of data? How content is different from format of data? Isn't the same?

Reply:
Could not find a reasonable - easy to understand (not too formal / scientific) definition of what schema is (in our context). Pls replace it with anything you deem reasonable and precise.